src/lista/repeticao/Ex67.java

```diff
@@ -5,20 +5,23 @@
 public class Ex67 {
     public static void main(String[] args) {
         Scanner leia = new Scanner(System.in);
-        int vetor[] = new int[5];
-        int soma = 0, i;
+        int numeros = 0, soma = 0;
 
-        for (i = 0; i < 5; i++) {
-            System.out.print("Digite seus numero para soma: ");
-            vetor[i] = leia.nextInt();
-            if (vetor[i] == 0) {
-                return;
+        System.out.print("Digite seus numero para a soma: ");
+        numeros = leia.nextInt();
+
+        do {
+            if (numeros == -1) {
```

Comment:
Se for digitado um número menor que -1, seu programa vai somar os valores? No exercício, o enunciado pede que sejam somados valores inteiros positivos.